packages/sync/src/Functions.php

```diff
@@ -102,12 +142,24 @@ public static function expand_synced_post_type( $sanitized_post_type, $post_type
 		return (object) $post_type_object;
 	}
 
+	/**
+	 * Returns site's post_type_features.
+	 *
+	 * @return mixed
```

Comment:
This is an array, according to WP core: https://opengrok.a8c.com/source/xref/trunk/wp-includes/post.php#1758

Reply:
f42036178